types/poisson-disk-sampling/OTHER_FILES.txt

```diff
@@ -0,0 +1,3 @@
+src/neighbourhood.d.ts
+src/sphere-random.d.ts
+src/tiny-ndarray.d.ts
```

Comment:
What are these files for? Are they a part of the API?

Reply:
They are part of it. However not directly added, since they can be used additionally/optionally as far as I understood the logic behind it.